--- a/deps/libsrtp/CMakeLists.txt
+++ b/deps/libsrtp/CMakeLists.txt
@@ -104,7 +104,6 @@ if(ENABLE_MBEDTLS)
   if(ENABLE_OPENSSL OR ENABLE_NSS)
     message(FATAL_ERROR "ssl conflict. can not enable mbedtls and openssl or nss simultaneously.")
   endif()
-  find_package(MbedTLS REQUIRED)
   set(MBEDTLS ${ENABLE_MBEDTLS} CACHE BOOL INTERNAL)
   set(GCM ${ENABLE_MBEDTLS} CACHE BOOL INTERNAL)
 endif()
